prow/cmd/deck/static/plugin-help-script.js

```diff
@@ -35,19 +35,19 @@ document.addEventListener("DOMContentLoaded", function(event) {
 });
 
 function configure() {
-    if(typeof branding === undefined){
+    if(typeof branding === 'undefined'){
```

Comment:
oh no

Reply:
@cjwagner one of our JS wizards mentioned that if `branding` can be both `null` and `undefined`, this will only catch one of the two cases and still cause a runtime failure -- suggested to use `if (!branding)` instead?

Reply:
Sure, I'll make a PR.